template/src/services/networking/axios.js

```diff
@@ -0,0 +1,82 @@
+import axios from 'axios'
+import {setToken, TokenType} from '.'
+import {REFRESH_TOKEN_KEY, TOKEN_KEY, UNAUTHORIZED} from '../../constants'
+import {getRefreshToken} from '../api'
+import {clearAllKeys, getString, setData} from '../../services/mmkv/storage'
+import {replace} from '../../navigation/NavigationService'
```

Comment:
'replace' is declared but its value is never read

Reply:
done